semcore/utils/CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 CHANGELOG.md standards are inspired by [keepachangelog.com](https://keepachangelog.com/en/1.0.0/).
 
+## [3.33.1] - 2022-06-16
```

Comment:
I suppose it is a minor change 34 rather than patch change 1

Reply:
Yeah! Thanks!